companies/bigpoint.json

```diff
@@ -0,0 +1,40 @@
+{
+    "slug": "bigpoint",
+    "relevant-countries": [
+        "bg",
+        "cz",
+        "dk",
+        "de",
+        "gr",
+        "gb",
+        "es",
+        "fi",
+        "fr",
+        "hu",
+        "it",
+        "nl",
+        "no",
+        "pl",
+        "pt",
+        "ro",
+        "sk",
+        "se"
+    ],
+    "categories": [
+        "entertainment"
+    ],
+    "name": "Bigpoint GmbH",
+    "address": "Drehbahn 47-48\n20354 Hamburg\nDeutschland",
+    "phone": "+49 40 8814130",
+    "fax": "+49 40 881413 906",
```

Comment:
I've probably overlooked something but where did you find that fax number?

Reply:
[here](https://www.bigpoint.net/imprint/). That was before I updated the `sources`. It is generic fax, not a privacy related, so I guess the fax can be removed.